persist/src/main/java/com/codeforcommunity/database/tableImpl/StubCommentTableImpl.java

```diff
@@ -62,10 +63,76 @@ public StubCommentTableImpl() {
 
   @Override
   public void saveComment(CommentRecord comment) {
+    // Once we start using the database, these operations will be handled for us.
+    comment.setId(this.getNextId(comment.getPostId()) + 1);
```

Comment:
We shouldn't add 1 if we're giving the next id now

Reply:
Whoops, missed that one